pkg/kudoctl/cmd/operator_verify.go

```diff
@@ -20,19 +25,57 @@ func newOperatorVerifyCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 		Short:   "verify operator parameters",
 		Example: "  kubectl kudo operator verify",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			//list.home = Settings.Home
-			return list.run(fs)
+			if err := validateOperatorArg(args); err != nil {
+				return err
+			}
+			return list.run(fs, args[0])
 		},
 	}
 
 	return cmd
 }
 
-func (c *operatorVerifyCmd) run(fs afero.Fs) error {
+func (c *operatorVerifyCmd) run(fs afero.Fs, path string) error {
+
+	pf, err := packages.FromFolder(fs, path)
+	if err != nil {
+		return err
+	}
+	warnings, errors := verify.Parameters(pf.Params)
+
+	if warnings == nil && errors == nil {
+		fmt.Fprintf(c.out, "operator is valid\n")
+	}
+	if errors != nil {
+		printErrors(c.out, errors)
+		//todo: return an error
+		return nil
+	}
+	if warnings != nil {
+		printWarnings(c.out, warnings)
+		return nil
+	}
```

Comment:
this block reads confusingly for me. can line 47 be pulled out past both of these nil checks, since they return nil, and print that it's valid otherwise?

Reply:
yeah... I had similar thoughts... but we can't determine validity without the double check of nils :(

Reply:
by the end of line 57 you'd know that both errors == nil and warnings == nil wouldn't you?

Reply:
oh... wait... I wanted errors and warnings to both print... your comment would make sense if we do return nil (like it currently does)... but I wanted all warnings and errors if available.

Reply:
just since you have return statements in those blocks